### DISCUSSION TITLE
fyne.io/fyne/v2/internal/preferences.go The implementation here is for reference only？

### DISCUSSION THREAD

**Discussion by junmaqiang** [fyne-io/fyne#3589] [Q&A] [answered]

**Comment by Jacalz** [comment, marked as answer]
That package is used internally for handling the preferences but is not there for reference only. A few other places inside the Fyne codebase imports that and uses it. However, it can not be used by projects outside of Fyne as that is not how the API is supposed to be used. Take a look at https://developer.fyne.io/explore/preferences how to use the preferences API.

**Reply by junmaqiang** [reply]
thank you
